<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-TK2O5BT" failures="1" tests="2" name="Default test(failed)" time="0.103" errors="0" timestamp="2020-05-01T12:06:32 CEST">
  <testcase classname="demo.TestNGRetryFailedDemo" name="test2" time="0.015"/>
  <testcase classname="demo.TestNGRetryFailedDemo" name="test3" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at demo.TestNGRetryFailedDemo.test3(TestNGRetryFailedDemo.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="demo.TestNGRetryFailedDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
